money_bot/utils/models.py

```diff
@@ -23,6 +23,7 @@ class User(Document):
     last_name = fields.StringField(allow_none=True)
     username = fields.StringField(allow_none=True)
     current_task_id = fields.IntegerField(default=-1)
+    referrer_id = fields.IntegerField()
```

Comment:
You should set default value to -1, because now you set it in code

Reply:
no, it's part of the structure, i've already explained u in real life